addons/website_project_issue/controllers/main.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import werkzeug
+
+from openerp import http
+from openerp.http import request
+
+from openerp.addons.website_portal.controllers.main import website_account
+
+class WebsiteAccount(website_account):
+    @http.route(['/my', '/my/home'], type='http', auth="user", website=True)
+    def account(self):
+        response = super(WebsiteAccount, self).account()
+        project_issues = request.env['project.issue'].search([])
+        response.qcontext.update({
+            'issues': project_issues
+        })
+        return response
+
+class website_project_issues(http.Controller):
+    @http.route(['/my/issues/<model("project.issue"):issue>'], type='http', auth="user", website=True)
+    def issues_followup(self, issue=None):
+        has_timesheet_enabled = issue._fields.get('timesheet_ids')
```

Comment:
@rim-odoo , @dmo-odoo 

website_project_issue is not depend on project_issue_sheet module. so either move timesheet_ids field into project_issue module or add project_issue_sheet in depend list of website_project_issue.

I propose to add project_issue_sheet in depend list of website_project_issue. what you think ?